File: 0136-single-number/0136-single-number.cs
public class Solution {
    public int SingleNumber(int[] nums) {
        var result = 0;
        if(nums.Length <= 1)
        {
            return nums[0];
        }

        for(int i=0; i<nums.Length; i++)
        {
           result ^= nums[i];
        }
        return result;

        


    }
}

File: 0136-single-number/0136-single-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    let result = 0;
    for(var i=0; i<nums.length; i++)
    {
        result ^= nums[i];
    }
    return result;
    
};